File: Leet-code/easy/9. Palindrome Number.php
class Solution {

/**
 * @param Integer $x
 * @return Boolean
 */
function isPalindrome($x) {

    $number = floor($x);
    $remainder = 0;
    $reverse = 0;

    if($number <=0) return false;


    while(floor($number) ){

        $remainder = $number % 10;
        $reverse = $reverse * 10 + $remainder;

        $number =floor($number /10);

    }
    return $x == $reverse;
}
}

File: Interviewbits/9. Palindrome Number.c


bool isPalindrome(int x){
    
    long long int reverse = 0;
    int remainder = 0;
    int num = x;

    if(num < 0) return false;

    while( num > 0 ){
        remainder = num % 10;
        reverse   = reverse * 10 + remainder;
        num      /= 10;
    }
    return reverse == x ;
}